cleancode/src/main/java/com/epam/cleancode/singleresponsibility/Calculator.java

```diff
@@ -25,6 +26,7 @@ private int getProduct(String numbers) {
         for (String s : numbers.split(SEPARATOR))
```

Comment:
{} have to be everywhere even in case when it is not mandatory but language specification (for eg when we have 1 line of code in if/for etc)